### PR TITLE
Added new parameter  to all execute method. Now it's possible to turn off statement preparation

### PR DESCRIPTION
**Pull request by chandr-andr**

### CODE REVIEW

File: python/psqlpy/_internal/__init__.pyi [hunk 1]
```diff
@@ -321,6 +321,7 @@ class Transaction:
         self: Self,
         querystring: str,
         parameters: list[Any] | None = None,
+        prepared: bool = True,
```
Comment (insani7y) [inline python/psqlpy/_internal/__init__.pyi:324]: You sure, we want to use prepared statements by default?

Comment (insani7y) [inline python/psqlpy/_internal/__init__.pyi:324]: Also, i would move this setting onto the pool definition
I think, you either want to use prep stmts everywhere, or nowhere
This is discussable

Comment (chandr-andr) [reply python/psqlpy/_internal/__init__.pyi:324]: I think so cuz I assume that most people don't use PGBouncer which can break the prepared statement.
Plus, if someone uses PGBouncer only 2/3 pool strategies can break this behavior.

And at the same time prepared statements lead to a significant performance boost.

Comment (chandr-andr) [reply python/psqlpy/_internal/__init__.pyi:324]: I don't think so, the prepared statement can be used to prepare `SELECT, INSERT, UPDATE, DELETE, MERGE, or VALUES` statements, there are a lot of other stmts, like CREATE or DROP which cannot be executed.

So the user would set global `prepared = False` only for one CREATE or DROP statement for the whole service.

**Pull request merged**